gazebo_plugins/test/test_gazebo_ros_diff_drive.cpp

```diff
@@ -126,7 +126,7 @@ INSTANTIATE_TEST_CASE_P(
   GazeboRosDiffDrive, GazeboRosDiffDriveTest, ::testing::Values(
     TestParams({"worlds/gazebo_ros_diff_drive.world"}),
     TestParams({"worlds/gazebo_ros_skid_steer_drive.world"})
-));
+),);
```

Comment:
Looks like the linters don't like this change.

Reply:
should be addressed in 90c45fc